manifests/y/yuma14/ShutdownCountdown/1.1.0/yuma14.ShutdownCountdown.locale.ja-JP.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: yuma14.ShutdownCountdown
+PackageVersion: 1.1.0
+PackageLocale: ja-JP
+Publisher: yuma14
+PublisherUrl: https://github.com/yuma140902/ShutdownCountdown
+Author: yuma140902
+PackageName: ShutdownCountdown
+PackageUrl: https://github.com/yuma140902/ShutdownCountdown
+License: MPL-2.0
+LicenseUrl: https://github.com/yuma140902/ShutdownCountdown/blob/master/LICENSE
+ShortDescription: 指定した時間にPCをシャットダウンする
+Moniker: shutdowncountdown
+ReleaseNotesUrl: https://github.com/yuma140902/ShutdownCountdown/releases

```

```suggestion
ReleaseNotesUrl: https://github.com/yuma140902/ShutdownCountdown/releases/tag/v1.1.0
```
Should point to the specific page for the version